packages/las-sdk-core/src/client.ts

```diff
@@ -538,11 +579,12 @@ export class Client {
    * @param deleteDocuments Set to true to delete documents in batch before deleting batch
    * @returns Batch response from REST API
    */
-  async deleteBatch(batchId: string, deleteDocuments: boolean = false): Promise<Batch> {
+  async deleteBatch(batchId: string, deleteDocuments = false): Promise<Batch> {
```

Comment:
Don't need type for boolean?

Reply:
By setting a default value, the type is inferred